multiqc/modules/goleft_indexcov/goleft_indexcov.py

```diff
@@ -0,0 +1,144 @@
+            header = fn['f'].readline()
+            sample_names = [self._clean_name(x) for x in header.strip().split()[2:]]
+            for parts in (l.rstrip().split() for l in fn['f']):
+                chrom, cov = parts[:2]
+                sample_vals = parts[2:]
+                if self._short_chrom(chrom) is not None:
+                    for val, sample in zip(sample_vals, sample_names):
+                        data[chrom][sample][float(cov)] = float(val)
+        if data:
+            chroms = sorted(data.keys(), key=self._short_chrom)
+            pconfig = {
+                'id': 'goleft_indexcov-roc-plot',
+                'title': 'ROC: genome coverage per scaled depth by chromosome',
+                'xlab': 'Scaled coverage',
+                'ylab': 'Proportion of regions covered',
+                'ymin': 0, 'ymax': 1.0,
+                'xmin': 0, 'xmax': 1.5,
+                'data_labels': [{"name": self._short_chrom(c)} for c in chroms]}
+            return desc + linegraph.plot([data[c] for c in chroms], pconfig)
+
+    def _clean_name(self, name):
+        """Remove standard extra extensions from sample names.
+        """
+        return name.replace(".bam", "")
+
+    def bin_plot(self):
+        desc = '<p>This plot identifies problematic samples using binned coverage distributions. \n\
+        We expect bins to be around 1, so deviations from this indicate problems. \n\
+        Low coverage bins (< 0.15) on the x-axis have regions with low or missing coverage. \n\
+        Higher values indicate truncated BAM files or missing data. \n\
+        Bins with skewed distributions (<0.85 or >1.15) on the y-axis detect dosage bias. \n\
+        Large values on the y-axis are likely to impact CNV and structural variant calling. \n\
+        See the \n\
+        <a href="https://github.com/brentp/goleft/blob/master/docs/indexcov/help-bin.md" target="_blank">goleft indexcov bin documentation</a> \n\
+        for more details.</p>'
+
+        data = {}
+        for fn in self.find_log_files(config.sp["goleft_indexcov"]["ped"], filehandles=True):
+            header = fn['f'].readline()[1:].strip().split("\t")
+            for sample_parts in (l.split("\t") for l in fn['f']):
+                cur = dict(zip(header, sample_parts))
+                cur["sample_id"] = self._clean_name(cur["sample_id"])
+                total = float(cur["bins.in"]) + float(cur["bins.out"])
+                data[cur["sample_id"]] =  {"x": float(cur["bins.lo"]) / total,
+                                           "y": float(cur["bins.out"]) / total}
+        if data:
+            pconfig = {
+                'id': 'goleft_indexcov-bin-plot',
+                'title': 'Problematic low and non-uniform coverage bins',
+                'xlab': 'Proportion of bins with depth < 0.15',
+                'ylab': 'Proportion of bins with depth outside of (0.85, 1.15)'}
```

Comment:
Now that we have proportions, can we set axis limits to 0 and 1?

<table>
<thead><tr><th>Without limits</th><th>With limits</th></tr></thead>
<tbody><tr><td>
<img src="https://cloud.githubusercontent.com/assets/465550/23219538/9c918a96-f91f-11e6-9f3e-ec505d12b4db.png">
</td><td>
<img src="https://cloud.githubusercontent.com/assets/465550/23219548/a4cc3ecc-f91f-11e6-88f1-584169655c0e.png">
</td></tr></tbody></table>

Failing that, could be good to use `yCeiling` and `yFloor` so that the plot automatic scaling doesn't go beyond possible values.